src/pages/[platform]/index.tsx

```diff
@@ -69,7 +80,42 @@ const PlatformOverview = ({ platform }) => {
           other resources will help you build, connect, and host fullstack apps
           on AWS. Get started by selecting your preferred framework.
         </Text>
+
         <FrameworkGrid currentKey={platform} />
+        <LinkCardCollection>
```

Comment:
Is this the exact same as what is being added to `src/pages/index.tsx`?  Can we move this into its own component and instead just import the component on the pages where it will be used?

Reply:
sure